scrapers/GASM.yml

```diff
@@ -0,0 +1,22 @@
+name: GASM
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - gasm.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //div[@class="gqText"]/span[@class="gqTitle" and 2]
```

Comment:
What is the `and 2` supposed to do? Afaik it will always be true?

Reply:
Nothing it does nothing and I don't even know why its there, I was working on something else so might have transposed it maybe lol.